pmda/util.py

```diff
@@ -223,6 +256,19 @@ def second_order_moments(S1, S2):
     return S
 
 
-def sumofsquares(array):
-    """Calculates the sum of squares"""
-    return np.sum((array - array.mean(axis=0))**2, axis=0)
+def fold_second_order_moments(*args):
+    """
+    'Fold' action for second_order_moments calculation.
+
+    Notes
+    -----
+    Example reduce function:
```

Comment:
Not sure if this is needed. You can just link to wikipedia pages for "reduce" (if there's one) or similar.

More important is letting a user understand what this does and what the inputs are.